internal/backends/gl/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    let dlopen = std::env::var("RUST_FONTCONFIG_DLOPEN").is_ok();
+    if dlopen {
+        println!("cargo:rustc-cfg=feature=\"fontconfig-dlopen\"");
```

Comment:
This won't work. It's not possible to change the features from a build script.
Also build script are run after the dependencies nare build.

This logic would have to go in the build script of yes-confondus
fontconfig

Reply:
It does work, but it requires this in the build.rs of both yeslogic-fontconfig-sys and here because both have conditional compilation depending on a Cargo feature. You are correct that Cargo determines dependencies before build.rs is run, which means that the feature activated here cannot enable an optional dependency: https://github.com/yeslogic/fontconfig-rs/pull/12#discussion_r808546342